utilities/installer/SAMPLE_host.cfg

```diff
@@ -15,11 +15,23 @@
 # Replace the < > with your information
 
 HOSTS:
-    <agent1>:
+    <IOS XR node fully qualified domain name>:
```

Comment:
Are we required to use FQDN or is that optional? Just wondering about test devices that don't have a DNS entry.